scene/gui/control.cpp

```diff
@@ -1840,53 +1840,76 @@ Rect2 Control::get_anchorable_rect() const {
 
 void Control::add_icon_override(const StringName &p_name, const Ref<Texture> &p_icon) {
 
-	ERR_FAIL_COND(p_icon.is_null());
+	// ERR_FAIL_COND(p_icon.is_null());
```

Comment:
You can remove those instead of commenting them out.

Reply:
Oh. Just thought commenting those was some kind of a style :man_facepalming: